war/src/main/js/plugin-manager-ui.js

```diff
@@ -15,6 +15,11 @@ function applyFilter(searchQuery) {
       var selectedPlugins = [];
 
       var filterInput = document.getElementById("filter-box");
+      filterInput.addEventListener("keypress", (event) => {
+        if (event.key === "Enter") {
+          event.preventDefault();
+        }
+      });
```

Comment:
is this still needed if you've moved the search box out of the form?

Reply:
Yes, this only control `Available` and `Installed`.